train_seq_elmo.py

```diff
@@ -142,9 +142,9 @@
         logger.info('############')
         logger.info('Epoch: {}'.format(indexs))
         pw.nvidia_memory_map()
-
+        iterator = train_dataset.get_tqdm(device)
         seq_model.train()
-        for f_c, f_p, b_c, b_p, flm_w, blm_w, blm_ind, f_w, f_y, f_y_m, _ in iterator = train_dataset.get_tqdm(device):
+        for f_c, f_p, b_c, b_p, flm_w, blm_w, blm_ind, f_w, f_y, f_y_m, _ in iterator:
```

Comment:
Nice catch!
I would prefer to change it to `in train_dataset.get_tqdm(device)`

Reply:
I will do it~

Reply:
`for f_c, f_p, b_c, b_p, flm_w, blm_w, blm_ind, f_w, f_y, f_y_m, _ in iterator in train_dataset.get_tqdm(device):`

error